Dockerfile

```diff
@@ -1,15 +1,37 @@
-FROM golang:alpine
+ARG GO_VERSION=1.17
+ARG ALPINE_VERSION=3.15
+
+# Build stage
+FROM golang:${GO_VERSION}-alpine${ALPINE_VERSION} as build
 
 WORKDIR /app
 
-COPY go.mod .
-COPY go.sum .
+COPY go.mod go.sum .
```

Comment:
`When using COPY with more than one source file, the destination must be a directory and end with a /`

So this just needs to be changed to `COPY go.mod go.sum ./`

Reply:
I can change it if you want, but this doesn't cause issues.

Reply:
It does when I try to build the image. Allow me to clarify:

```
$ docker-compose up
Step 1/17 : ARG GO_VERSION=1.17
...
Step 5/17 : COPY go.mod go.sum .
When using COPY with more than one source file, the destination must be a directory and end with a /
ERROR: Service 'portal' failed to build : Build failed
```

Docker 2.10.10

Reply:
Ah.... no issues with Docker 2.10.11. But I think that's because I'm using [buildx](https://docs.docker.com/buildx/working-with-buildx/). So this will need to be changed indeed.